wandb/proto/wandb_internal.proto

```diff
@@ -150,6 +151,25 @@ message GitRepoRecord {
   string commit = 2;
 }
 
+// Path within nested configuration dictionary.
+
+// Used to specify which paths within various config objects should be filtered
+// in or out of job inputs.
+message ConfigFilterPath {
+  repeated string path = 1;
+}
+
+// Specifies filter mode and paths for setting job inputs from wandb.config.
+//
+// If this record is published to the core internal process then it will filter
+// the given paths into or out of the job inputs it builds. If the exclude field
+// is true then the paths will be excluded from the job inputs, otherwise they
+// will be included.
+message LaunchWandbConfigParametersRecord {
+  repeated ConfigFilterPath paths = 1;
+  bool exclude = 2;
```

Comment:
Why not have separate include paths and exclude paths? It would also help to clarify that these are prefixes, not full paths.

```suggestion
  repeated ConfigFilterPath include_prefixes = 1;
  repeated ConfigFilterPath exclude_prefixes = 1;
```

Reply:
Yeah I had it like this before and then changed it to make it more clear that these arguments were mutually exclusive. But now that they are compatible I will change this back